.changeset/tall-meals-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-events-backend': patch
+---
+
+Export `InMemoryEventBroker` to allow decoupling of the catalog and events backends in the `example-backend`.
```

Comment:
A bit more information for people to migrate to the new approach could help. Maybe the upgrade helper diff is enough, though 🤔

Reply:
Where would be the best place to document that?

Reply:
I found a place. I updated the readme file.